apps/aws-notebook/manifest.yaml

```diff
@@ -0,0 +1,9 @@
+daemons:
+  openvpn:
+    run:
+      cmd: |
+        set -ex
+        sudo service dbus start
```

Comment:
Please do not mix another background task in the same daemon. Define a separate daemon, like `dbus`